### ISSUE TITLE
Can't open "New Window" in separate WM managed window on OSX 10.13.6 High Sierra

### ISSUE THREAD

**Issue by dossy** [macvim-dev/macvim#758]
Installed MacVim via Homebrew cask:

```
VIM - Vi IMproved 8.1 (2018 May 18, compiled Aug 14 2018 05:28:49)
macOS version
Included patches: 1-280
Compiled by travis@Traviss-Mac-1044.local
```

Using Cmd-N to open a New Window creates a new tab inside the existing window as a tab, not as a separate GUI window.  This error is written to the console at the same time:

```
Oct  9 12:15:53 Serenity com.apple.xpc.launchd[1] (com.apple.xpc.launchd.domain.user.501): Service "com.apple.xpc.launchd.unmanaged.Vim.22193" tried to register for endpoint "/applications/macvim.app.vim" already registered by owner: com.apple.xpc.launchd.unmanaged.Vim.22164
```

I can click-and-drag the window "tab" from the window and pull it out into its own separate GUI window.

These window tabs can have their own tabpages (which I can create with Cmd-T or :tabnew).  It's as if the newly created window was created as an OS-managed tab into the parent MacVim GUI window, instead of its own standalone GUI window.

Here's a screenshot of the symptom:

![image](https://user-images.githubusercontent.com/65378/46683367-0b620c80-cbbe-11e8-98db-ac7113fb7135.png)

**Comment by dossy** [comment]
Update: I found the cause of this problem!

[Open folders in new window (high Sierra)](https://discussions.apple.com/thread/8090438)

The culprit is in "System Preferences > Dock > Prefer tabs when opening documents" - it was set to "Always" (and I'm not sure if that's the default setting or not) ... if I set it to "Manually" then I get back the New Window behavior I was expecting - both in MacVim and in Finder - and I'm betting it'll "fix" the problem in other places as well.

Going to close this issue, but man ... this was annoying as hell ...

**Comment by ychin** [comment]
@dossy I know this issue has been closed, but #788 will fix this properly. MacVim will disable native tabs so you won't see this weird mix of native tabs and Vim tabs again and feel free to set your "Prefer tabs when opening documents" preference back to whatever you want.

**Comment by dossy** [comment]
@ychin - Thanks for the update.
